packages/status/src/class-status.php

```diff
@@ -151,9 +151,8 @@ public function is_local_site() {
 		// Check for localhost and sites using an IP only first.
 		$is_local = site_url() && false === strpos( site_url(), '.' );
 
-		// @todo Remove function_exists when WP 5.5 is the minimum version.
-		// Use Core's environment check, if available.  Added in 5.5.0 / 5.5.1 (for `local` return value)
-		if ( function_exists( 'wp_get_environment_type' ) && 'local' === wp_get_environment_type() ) {
+		// Use Core's environment check, if available. Added in 5.5.0 / 5.5.1 (for `local` return value).
+		if ( 'local' === \wp_get_environment_type() ) {
```

Comment:
Looks like removing the `function_exists()` makes the tests for packages/status and packages/jitm unhappy, because the function doesn't exist during the package tests.

I see packages/status is already using something (`Brain\Monkey\Functions`) to mock the function in a different test. Perhaps that should be used for the others that are failing, and for packages/jitm's test.

Reply:
👍 409fc36c06d110a869b0844ee01962e92588c7f7 should take care of that.